pytorch_toolkit/action_recognition_2/model_templates/gesture_recognition/s3d-rgb-mobilenet-v3-stream-jester/template.yaml

```diff
@@ -0,0 +1,69 @@
+name: s3d-rgb-mobilenet-v3-stream-jester
+domain: Action Recognition 2
+problem: Gesture Recognition
+framework: OTEAction
+summary: ASL Recognition based on S3D-MobileNet-V3.
+annotation_format: CustomAction
+classes: []
```

Comment:
it is not documented, but it is not needed here in your case I guess

Reply:
Removed.